### ISSUE TITLE
[meta] login component in desktop

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/application-services#2557]
**TL;DR Why:** Integrate the logins rust component to desktop to update the current code base with more up to date, reliable, and easier to maintain code. The rust upgrade can also help us build on top of places at a quicker rate. Upgrading logins on desktop can help us introduce new features for users. Users could potentially save more than passwords, and could do more password management. 

The sync team can start working on the logins component once decisions around master passwords, and certain cryto questions have been made. These decisions will be made by the Logins team. For more information on the engineering work for introducing the logins component in desktop, please refer here: https://docs.google.com/document/d/1nGM3pc__OlxDQ4G5xAGZMfJjWNK6YDh00lEUEyJJx2s/edit#heading=h.f623jc1famd8

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/SYNC-767)
┆Sprint: Backlog

**Comment by data-sync-user** [comment]
➤ Ana Medinac commented:

Later. Dependent on external team and decisions around Master Password, and crypto work.

**Comment by jdragojevic** [comment]
Tracking as a jira story until we're ready to commit and break down the work.
